snap/snapcraft.yaml

```diff
@@ -37,3 +37,6 @@ parts:
     python-version: python3
     source: .
     source-type: git
+    process-dependency-links: true
+    python-packages:
+      - "git+https://github.com/leafty/python-future.git@fix-read-only#egg=future-0.16.0"
```

Comment:
1. Will you submit a PR to the upstream project?
2. Can you move the changes to `https://github.com/SwissDataScienceCenter/python-future.git`?

Reply:
1. Yes, wanted to know first if the fix works
2. Will do